### PR TITLE
Updating monitoring.md with Grafana Config Info

### LINKED ISSUE #1567: Update Grafana Documentation

### PR DESCRIPTION
**Pull request by robertandremitchell**
# PULL REQUEST

## Summary
This adds more detail to the `monitoring.md` to ensure users understand the role of Grafana in our logging/monitoring infrastructure, as well as which files to check/update in the event questions arise for configurations.

## Related Issue
Fixes #1567

## Additional Information
I've included @robertmitchellv and @m-goggins as reviewers for higher exposure.

## Checklist

- [ ] If this code affects the other scrum team, have they been notified? (In Slack, as reviewers, etc.)

[//]: # (PR title: Remember to name your PR descriptively!)

### CODE REVIEW

File: containers/orchestration/monitoring.md [hunk 1]
```diff
@@ -1,50 +1,62 @@
...
-* _Manual Instrumentation_: This type of instrumentataion must be configured and implemented by a developer. In any case where you'd like to collect and emit information that isn't automatically collected--or you'd like to customize how a particular type of information is emitted--manual instrumentation is required. To instrument a piece of an application, you need three things: a Provider, an Agent, and an Instrument. These things aren't hard to set up, but do require invoking in the correct order. 
+* _Manual Instrumentation_: This type of instrumentataion must be configured and implemented by a developer. In any case where you'd like to collect and emit information that isn't automatically collected--or you'd like to customize how a particular type of information is emitted--manual instrumentation is required. To instrument a piece of an application, you need three things: a Provider, an Agent, and an Instrument. These things aren't hard to set up, but do require invoking in the correct order.
```
Comment (bamader) [inline containers/orchestration/monitoring.md:24]: Just caught a typo of my own making that we might as well correct here: there's an extra 'a' at the end of instrumentation

**Review by bamader — APPROVED** [review]
LGTM!

**Comment by robertandremitchell** [comment]
@m-goggins , @robertmitchellv , low priority but if you have a chance to look.

@DanielPaseltiner , based on our convo yesterday, do you think we should add language about the importance of OTel and/or reframe the Prom/Graf sections as "this is the internal DIBBs standard"? Or is it fine as-is?

**Review by DanPaseltiner — APPROVED** [review]
This looks great. @robertandremitchell I think we should merge the documentation as is and think about bringing @mattmgoldberg in to help us craft an optimal approach here. We will also probably want a lighter weight version of this doc for the website for a more general audience.

File: containers/orchestration/monitoring.md [hunk 2]
```diff
@@ -1,50 +1,62 @@
...
 * We configure the OTel Collector with the listening/transmitting mechanisms we want to use, and then activate the Collector with a service pipeline. This is set up in `otel-collector-config.yaml`.
 * We configure Prometheus with a scrape job to fetch the metrics that the OTel Collector put together for us based on our instrumentation. This is configured in `prometheus.yml`.
-* We link Prometheus to Grafana as its database source so that it can pull metrics from Prometheus' local DB into visualization dashboards (TODO: ADD MORE INFO ON THIS ONCE WE DO IT).
+* We link Prometheus to Grafana as its database source so that it can pull metrics from Prometheus' local DB into visualization dashboards.
+* We configure `grafana/datasources/datasources.yml` to specify the Prometheus connection as the default DB to use for the visualization in Grafana.
+* We configure `grafana/dashboards/dashboard.yml` for the Dashboards we use throughout Grafana. New dashboards can be created in Grafana UI then downloaded as JSON and saved within the `grafana/dashboard` folder for use by any user logging into `localhost:4000`.
```
Comment (m-goggins) [inline containers/orchestration/monitoring.md:54]:
```suggestion
* We configure `grafana/dashboards/dashboard.yml` for the Dashboards we use throughout Grafana. New dashboards can be created in Grafana UI then downloaded as JSON and saved within the `grafana/dashboards` folder for use by any user logging into `localhost:4000`.
```

File: containers/orchestration/monitoring.md [hunk 3]
```diff
@@ -1,50 +1,62 @@
...
 * We spin up local instances of the OTel Collector, Prometheus, and Grafana, making sure to expose specific relative ports and endpoints. This is executed in the project's `docker-compose.yml`.
 * We configure the OTel Collector with the listening/transmitting mechanisms we want to use, and then activate the Collector with a service pipeline. This is set up in `otel-collector-config.yaml`.
 * We configure Prometheus with a scrape job to fetch the metrics that the OTel Collector put together for us based on our instrumentation. This is configured in `prometheus.yml`.
-* We link Prometheus to Grafana as its database source so that it can pull metrics from Prometheus' local DB into visualization dashboards (TODO: ADD MORE INFO ON THIS ONCE WE DO IT).
+* We link Prometheus to Grafana as its database source so that it can pull metrics from Prometheus' local DB into visualization dashboards.
+* We configure `grafana/datasources/datasources.yml` to specify the Prometheus connection as the default DB to use for the visualization in Grafana.
```
Comment (m-goggins) [inline containers/orchestration/monitoring.md:53]:
```suggestion
* We configure `grafana/datasources/datasources.yml` to specify Prometheus connection as the default data source to use for the visualization in Grafana.
```

just to match the language further down in the document

File: containers/orchestration/monitoring.md [hunk 4]
```diff
@@ -59,32 +71,59 @@ When an API request hits the `process-message` endpoint of the Orchestration Ser
...
+* `paths` defines the directory paths for Grafana components, such as where it stores data, logs, plugins, and provisioning.
+* `server` defines settings related to the Grafana webserver, including the port, domain name, and public URL.
+* `database` is the configuration for Grafana's own database, which is separate from Prometheus' database.
+* `log` controls the logging mechanism. Right now, logs are output to console and at the info level.
+* `security` settings is where the admin_user and admin_password can be specified.
+* `users` are user-specific settings. Right now, we allow users to sign-in and create accounts, but we may want to revisit if we want to create a user hierarchy (admin, read-only user, etc.).
+
+#### `grafana/dashboards/dashboard.yml`
+
+This is a simple yaml file that specifies that dashboards saved here should be pulled in and loaded to our Grafana UI. New dashboards can be created in the UI then downloaded and saved here as JSON.
```
Comment (m-goggins) [inline containers/orchestration/monitoring.md:125]:
```suggestion
This is a simple yaml file that specifies that dashboards saved here should be pulled in and loaded to the Grafana UI. New dashboards can be created in the UI then downloaded and saved here as JSON.
```

**Review by m-goggins — APPROVED** [review]
Looks good to me! Just added some minor edits

**Pull request merged**